### DISCUSSION TITLE
zh/posts/%E8%BD%AF%E4%BB%B6%E8%AE%BE%E8%AE%A1%E9%87%8D%E7%82%B9

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#470] [Announcements]
# zh/posts/%E8%BD%AF%E4%BB%B6%E8%AE%BE%E8%AE%A1%E9%87%8D%E7%82%B9

"" 第一章 1.[填空题] 软件工程层次图： 工具 方法 过程 质量关注点 2.从分析模型到设计模型的转化，分析模型都有： 基于场景的元素 面向流的元素 基于类的元素 行为元素 3.质量指导原则： 设计应使用具备良好设计特征的构件组成 设计应模块化 设计应包含数据，接口，构件，体系结构的明确表示 设计应导出数据结构 设计应导出独立的构件 设计应导出接口 设计应导出获取的信息 设计应使用能有效传达其意义的表示法来表达

https://erickiku.github.io/zh/posts/%E8%BD%AF%E4%BB%B6%E8%AE%BE%E8%AE%A1%E9%87%8D%E7%82%B9.html

**Comment by EricKiku** [comment]
这能背会吗，靠

**Reply by Mister-Hope** [reply]
## DO NOT USE MY REPO IN PRODUCTION!

**Reply by EricKiku** [reply]
>I'm sorry, my mistake
